### ISSUE TITLE
Failed to package strapi

### ISSUE THREAD

**Issue by Lassulus** [nix-community/dream2nix#69]
after checking out dreampkgs I ran the following inside the repo
commands:
```
nix run github:davhau/dream2nix add github:strapi/strapi/v4.0.2
nix run .#strapi
```
error:
```
warning: Git tree '/home/lass/src/dreampkgs' is dirty
error: builder for '/nix/store/bvscf68p8f1467z188zji5fw7yc858wi-strapi-unknown.drv' failed with exit code 1;
       last 8 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/6i2ljff2rgkxwm9qfshv4s3hfblxgdjw-source
       > patching sources
       > d2nPatchPhase
       > Traceback (most recent call last):
       >   File "/nix/store/skybcqxgqssjn28naxd48p5h42ngsbji-fix-package.py", line 34, in <module>
       >     if version not in ["unknown", package_json['version']]:
       > KeyError: 'version'
       For full logs, run 'nix log /nix/store/bvscf68p8f1467z188zji5fw7yc858wi-strapi-unknown.drv'.
```

I couldn't find a way to run dream2nix directly without dreampkg. If someone could tell me how to do that I could try a more recent version of dream2nix probably?

**Comment by Lassulus** [comment]
ok, the new version throws a new error:
```
this derivation will be built:
  /nix/store/4nfqvqnhha4k6dbjlhdhbkr9x6jzs7s5-strapi-unknown.drv
building '/nix/store/4nfqvqnhha4k6dbjlhdhbkr9x6jzs7s5-strapi-unknown.drv'...
unpacking sources
unpacking source archive /nix/store/i9vabgbdr2ml3r2az7q0sys2l96b8l45-strapi
patching sources
d2nLoadFuncsPhase
d2nPatchPhase
configuring
installing: @babel/polyfill
Traceback (most recent call last):
  File "/nix/store/9vqgazzikzpzc6c5l9sys06rad9q5ysn-install-deps.py", line 162, in <module>
    add_to_bin_path = install_direct_dependencies()
  File "/nix/store/9vqgazzikzpzc6c5l9sys06rad9q5ysn-install-deps.py", line 48, in install_direct_dependencies
    os.symlink(origin, f"{root}/{module}/{submodule}")
FileExistsError: [Errno 17] File exists: '/nix/store/va8lj4aslqxqqrbzyp1nvq2lv379w719-__at__babel__slash__polyfill-7.12.1/lib/node_modules/@babel/polyfill' -> '/nix/store/ihrv7mwrpm4dwk1fsacxak7ijxjh19lh-strapi-unknown/lib/node_modules/strapi/node_modules/@babel/polyfill'
```

**Comment by Lassulus** [comment]
fixed by #70
